Source/WebKit/NetworkProcess/Downloads/cocoa/DownloadCocoa.mm

```diff
@@ -64,15 +66,44 @@
     m_downloadTask.get()._pathToDownloadTaskFile = path;
 
     [m_downloadTask resume];
+
+#if HAVE(MODERN_DOWNLOADPROGRESS)
+    if (RetainPtr<NSData> placeholderURLBookmark = [dictionary objectForKey:@"ResumePlaceholderURLBookmarkData"]) {
+        RetainPtr nsActivityAccessToken = toNSData(activityAccessToken);
+        RetainPtr pathString  = adoptNS([[NSString alloc] initWithUTF8String:WTF::FileSystemImpl::fileSystemRepresentation(path).data()]);
+        RetainPtr destinationURL = adoptNS([[NSURL alloc] initFileURLWithPath:pathString.get() isDirectory:NO]);
+
+        BOOL bookmarkDataIsStale = NO;
+        NSError *bookmarkResolvingError = nil;
+        RetainPtr placeholderURL = adoptNS([[NSURL alloc] initByResolvingBookmarkData:placeholderURLBookmark.get() options:0 relativeToURL:nil bookmarkDataIsStale:&bookmarkDataIsStale error:&bookmarkResolvingError]);
+        BOOL usingSecurityScopedURL = [placeholderURL startAccessingSecurityScopedResource];
+
+        if (placeholderURL) {
+            m_progress = adoptNS([[WKModernDownloadProgress alloc] initWithDownloadTask:m_downloadTask.get() download:*this URL:destinationURL.get() useDownloadPlaceholder:YES resumePlaceholderURL:placeholderURL.get() liveActivityAccessToken:nsActivityAccessToken.get()]);
+            startUpdatingProgress();
+        } else
+            RELEASE_LOG(Network, "Unable to create resume placeholder URL, error = %@", bookmarkResolvingError);
```

Comment:
Seems like this should be a RELEASE_LOG_ERROR. Also, The logging line would ideally start with "Download::resume: " as per our usual pattern.

Reply:
Thanks for reviewing! Fixed in latest patch.